deploy/install.sh

```diff
@@ -106,60 +106,93 @@ applyDynatraceOperator() {
 }
 
 applyDynaKubeCR() {
-  if [ -z "$CLUSTER_NAME" ]; then
-    cat <<EOF | "${CLI}" apply -f -
-apiVersion: dynatrace.com/v1alpha1
-kind: DynaKube
-metadata:
-  name: dynakube
-  namespace: dynatrace
-spec:
-  apiUrl: ${API_URL}
-  skipCertCheck: ${SKIP_CERT_CHECK}
-  kubernetesMonitoring:
-    enabled: true
-  routing:
-    enabled: true
-  classicFullStack:
-    enabled: true
-    tolerations:
-    - effect: NoSchedule
-      key: node-role.kubernetes.io/master
-      operator: Exists
-    env:
-    - name: ONEAGENT_ENABLE_VOLUME_STORAGE
-      value: "${ENABLE_VOLUME_STORAGE}"
+  dynakube=$(
+    cat <<EOF
```

Comment:
Not a big issue, but it's a bit weird that `cat <<EOF` is indented everywhere but the closing `EOF` isn't.
Makes reading the code a bit bumpy.

Reply:
You can't intend the closing EOF - otherwise you would loose indentation for the string above